File: Lesson7 String Algorithms/Problem1/Solutions/Solution_built_in.java
import java.util.Scanner;
public class Solution_built_in{
     public static void main(String []args){
        Scanner sc = new Scanner(System.in);
        String s = sc.next();
        if(s.contains("CCC")){
            System.out.println("NO");
        }
        else {
            System.out.println("YES");
        }
        sc.close();
     }
}

File: Lesson7 String Algorithms/Problem1/Solutions/Solution_built_in.cpp
#include <iostream>

using namespace std;

int main()
{
   string s; cin>>s;
   if(s.find("CCC") != string::npos){
       cout<<"NO"<<endl;
   }
   else{
       cout<<"YES"<<endl;
   }
   return 0;
}
